bedrock/teach/templates/teach/smarton/security.html

```diff
@@ -15,7 +15,12 @@
 <article class="main">
 
   {# Navbar macro. Params: Page title, tweet text, Twitter URL, Facebook URL, Google+ URL #}
-  {{ smarton_navbar(_('Cybersecurity'), _('Look both ways, stay updated, and make sure your passwords are as buff as you are. Get smart on cybersecurity.') + hashtag, 'http://mzl.la/cybersecurity', 'https://www.mozilla.org/teach/smarton/security/', 'https://www.mozilla.org/teach/smarton/security/') }}
+  {# see bug 1188591, comment 10 for details regarding the need for the en-US conditional #}
+  {% if LANG == 'en-US' %}
+    {{ smarton_navbar(_('Cybersecurity'), _('Stay updated, and make sure your passwords are as buff as you are. Get smart on cybersecurity.') + hashtag, 'http://mzl.la/cybersecurity', 'https://www.mozilla.org/teach/smarton/security/', 'https://www.mozilla.org/teach/smarton/security/') }}
```

Comment:
I was wondering about that as well, not entirely sure how to accomplish that, unless it is done the same way as the hashtag perhaps?